qubit-events-transaction.liquid

```diff
@@ -60,7 +60,7 @@
         }, 
         url: "{{ item.url }}",
         description: {{ item.product.description | strip_html | truncate: 2048 | json }},
-        category: [{{ item.product.type | json}}],
+        categories: [{{ item.product.type | json}}],
         images: [{{ item.image | json }}]
```

Comment:
@alexolivier is it actually `image` when it's an `item`, and `images` when it's a `product`?